### DISCUSSION TITLE
Faces don't seem to be scanned during indexing.

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#2600] [Q&A]
When uploading and indexing photo's. It doesn't add new faces to: http://photoprism:2342/people
I've i edit the metadata of the new photo's, in the persons tab the faces do show up.

What am I doing wrong?

**Comment by graciousgrey** [comment]
Only face clusters show up in the people section: https://docs.photoprism.app/user-guide/organize/people/#recognized-new-people.
To see all photos with unnamed faces search for faces:new
